opencti-platform/opencti-front/src/components/list_cards/ListCards.js

```diff
@@ -170,6 +170,7 @@ class ListCards extends Component {
           <div className={classes.filters}>
             {map((currentFilter) => {
               const label = `${truncate(t(`filter_${currentFilter[0]}`), 20)}`;
+              const localFilterMode = currentFilter[0].endsWith('not_eq') ? t('AND') : t('OR');
```

Comment:
Can be extracted in the utils file to avoid duplicated code (file ListCards, ListLines, ReportKnowledge, ect.)

Reply:
fix with the component FilterIconButton